SodaInventory/Controllers/InventoryController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SodaInventory.Model;
+
+namespace SodaInventory.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class InventoryController : ControllerBase
+    {
+        private readonly DatabaseContext _context;
+
+        public InventoryController(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/GetInventory
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<object>>> GetInventory()
+        { 
+            return await _context.ItemQuantities.Join(
+                _context.Items,
+                items => items.ItemId,
+                itemQty => itemQty.ItemId,
+                (itemQty, items) => new
+                {
+                    itemQty.ItemId,
+                    items.ItemName,
+                    itemQty.Amount,
+                    itemQty.ModerateLevel,
+                    itemQty.UrgentLevel,
+                    Uom = items.Units
+
+                }
+                )
+                .GroupBy(item => new { item.ItemId, item.Uom , item.ModerateLevel, item.UrgentLevel, item.ItemName})
+                .Select(item => new {
+                    ItemId = item.Key.ItemId ?? default,
+                    Amount = item.Sum(item => item.Amount),
```

Comment:
By summing the amounts, it could make it so that the moderate and urgent levels are never seen for the selected store as other stores would have what is needed.

Reply:
I added a storeId parameter. Now it should add them up and group them by stores.